### ISSUE TITLE
Warning: Missing dependency

### ISSUE THREAD

**Issue by oheinemann** [phoenixv/cachefilesd#3]
I get an warning that the "puppetlabs-stdlib" module is missing when installing other modules after this module.

In the [Modulefile](https://github.com/phoenixv/cachefilesd/blob/master/Modulefile) you write:
`dependency 'puppetlabs-stdlib', '>3.0.0'`

I guess it should be something like:
`dependency 'puppetlabs/stdlib', '>3.0.0'`

**Comment by phoenixv** [comment]
What version of puppet are you using, and what version of puppetlabs-stdlib
do you have? That syntax has been working until now, so something may have
changed in puppet. May be time for me to set up a test box and do some
updates.

Bob

Bob

I don't bother with WEP or WPA, I just got Bruce Schneier to autograph my
wireless access point.

**Comment by oheinemann** [comment]
I installed your module in a vagrant configuration with a `precise64` base box.
So the puppet version is `2.7.19` - sorry, not the freshest!

**Comment by phoenixv** [comment]
I've never tested it with a version that old, I'd recommend you find a way
to use a newer version as really old versions of puppet will not be your
only problem. _Serious_ security issues _will_ be present in your base box,
which will become free root access for anyone if you deploy that. This
issue concerning vagrant (old containers) is huge and should not be ignored.

That said if you care to make a patch I'll put it in, but I will not likely
make one as 3.7 is current and 4.0 is on the way.

It _might_ work with 2.8, I did some testing with that version when I wrote
it, but I do not remember if it worked or not. This may be one of the
modules that did not.

Bob

Bob

I don't bother with WEP or WPA, I just got Bruce Schneier to autograph my
wireless access point.

On Fri, May 29, 2015 at 11:44 AM, Oliver Heinemann <notifications@github.com

**Comment by oheinemann** [comment]
Yes, you're absolutely right. It's not the best idea to use an old version of puppet.

But, I just tested with another box (Debian Jessie) with `puppet 3.7.2` installed and i get the same errors.

So, I guess it's a typo in your [Modulefile](https://github.com/phoenixv/cachefilesd/blob/master/Modulefile).

Just take a look into the docs of puppet 3:
[https://github.com/puppetlabs/puppet-docs/blob/master/source/puppet/3/reference/modules_publishing.markdown#dependencies-in-the-modulefile](https://github.com/puppetlabs/puppet-docs/blob/master/source/puppet/3/reference/modules_publishing.markdown#dependencies-in-the-modulefile).

They wrote: "Warning: The full name in a dependency must use a slash between the username and module name."

I'm sorry if I bother you.
Olli
